automation/project/Dependencies.scala

```diff
@@ -4,13 +4,10 @@ import sbt.Keys._
 object Dependencies {
   val jacksonV = "2.9.0"
 
-  val serviceTestV = "0.12-7b3f0d6"
+  val serviceTestV = "0.13-dc0998e"
 
   val workbenchExclusions = Seq(
-    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-model_$scalaBinaryVersion"),
-    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-util_$scalaBinaryVersion"),
-    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-metrics_$scalaBinaryVersion"),
-    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-google_$scalaBinaryVersion")
+    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-metrics_2.12"),
```

Comment:
Two issues with this code ... 

First, `$scalaBinaryVersion` does not work here. when used in a string like this, it resolves to a value of `SettingKey(This / This / This / scalaBinaryVersion)`, so we are actually excluding artifacts with a name of e.g. `workbench-model_SettingKey(This / This / This / scalaBinaryVersion)`. This doesn't cause an *error*, but it does fail to exclude the artifacts we wanted to exclude.

Second, the code fails to compile due to missing classes if we actually exclude workbench-model, workbench-util, or workbench-google. Before this PR, since the exclusion rules were faulty, we weren't actually excluding anything, so compilation succeeded. Now that I've fixed the syntax, I need to also tweak what is included vs. excluded.